### ISSUE TITLE
Redundant if statement can be removed

### ISSUE THREAD

**Issue by annp89** [torchbox/wagtail#1763]
https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailusers/views/users.py#L54 checks to see if `ordering in ["name", "username"]` and https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailusers/views/users.py#L55 checks if `ordering != "name"` and filters users by `username` since that's the only other option.

Those lines can probably be refactored as follows:
```
if 'ordering' in request.GET:
    ordering = request.GET['ordering']
    if ordering == 'username':
        users = users.order_by(ordering)
```

If that's sounds reasonable, I'd be happy to open a pull request for the above mentioned changes.

**Comment by kaedroho** [comment]
Thanks, the new way definitely looks a clearer to me. Pull request welcome!

**Comment by annp89** [comment]
Yay, Thanks!! Will do shortly.

**Comment by alexgleason** [comment]
Looks like it's the same here, too: https://github.com/torchbox/wagtail/commit/46840b2c2f3d6546b9b6ec12cab561f361da2682#diff-61ccba6e8955e9ec6a7a15499d32b1a7R46

Might also want to change that line.

@davecranwell Are you cool with this change?

**Comment by annp89** [comment]
Will make the change mentioned above.

**Comment by alexgleason** [comment]
Hmm. In that case I think this makes more sense, then.

```python
# Ordering
try:
    ordering = request.GET['ordering']
except KeyError:
    ordering = None

if ordering not in ['title', '-created_at']:
    ordering = '-created_at'

documents = documents.order_by(ordering)
```

**Comment by annp89** [comment]
I think we simplify even further. Looks like the default is ```'-created_at'``` if ordering is not specified
```
# Ordering
ordering = '-created_at'
try:
    ordering = request.GET['ordering']
except KeyError:
    pass
documents = documents.order_by(ordering)
```

And i think in the pull request i have, we can probably use the same construct but the default there appears to be ```name``` . Thoughts?

**Comment by timheap** [comment]
Why not:
```
# Ordering
default_ordering = '-created_at'
allowed_orderings = ['title', '-created_at']

ordering = request.GET.get('ordering', default_ordering)
if ordering not in allowed_orderings:
    ordering = default_ordering
documents = documents.order_by(ordering)
```

**Comment by alexgleason** [comment]
@annp89 Only problem with that it doesn't validate the user input. This part: `if ordering not in ['title', '-created_at']:` is making sure the ordering is valid before passing it to documents.order_by() to prevent a runtime error.

@timheap Ahh, nice! I didn't know [`dict.get()`](https://docs.python.org/3/library/stdtypes.html#dict.get) allows for a default value, that simplifies it quite a bit.

**Comment by alexgleason** [comment]
You could also just omit the default value from it entirely, which causes `dict.get()` to default to `None`.

```python
# Ordering
default_ordering = '-created_at'
allowed_orderings = ['title', '-created_at']

ordering = request.GET.get('ordering')
if ordering not in allowed_orderings:
    ordering = default_ordering
documents = documents.order_by(ordering)
```

I think it's better for ordering to be set to `None` when the key doesn't exist, because it's valid to want to know that the key was never set. For instance, maybe in the future you'd rather pass `None` directly to the `order_by` function, and let it determine the default ordering based on a property defined as a part of the `documents` object.

**Comment by alexgleason** [comment]
@annp89 Do you want to update this? Or if you don't have time I can. Thanks!

**Comment by annp89** [comment]
^ go for it. Thanks for checking in.

**Comment by gasman** [comment]
Fixed by #1766 (but if there are further tweaks to be made, feel free to open as new PRs :-) )
